packages/mobile/src/services/offline-downloader/offline-sync.ts

```diff
@@ -156,32 +166,20 @@ export const syncCollectionTracks = async (
   // TODO: will discovery serve a removed playlist?
   if (!updatedCollection) return
 
-  if (
-    moment(updatedCollection.updated_at).isSameOrBefore(
-      offlineCollection.updated_at
-    )
-  ) {
-    return
-  }
-
-  downloadCollection(
-    updatedCollection,
-    isFavoritesDownload,
-    /* skipTracks */ true
-  )
-  if (updatedCollection.cover_art_sizes !== offlineCollection.cover_art_sizes) {
-    downloadCollectionCoverArt(updatedCollection)
-  }
-
-  const oldTrackIds = new Set(
-    offlineCollection.tracks?.map((track) => track.track_id)
-  )
+  const oldTrackIds = new Set([
+    ...(offlineCollection.tracks
```

Comment:
can we do the map filter separtely in a const, like offlineCollectionTrackIds?, then oldTrackIds is a bit easier to read? nit for sure